descriptions/css/properties/display.json

```diff
@@ -0,0 +1,9 @@
+{
+  "css": {
+    "properties": {
+      "display": {
+        "__short_description": "The <strong><code>display</code></strong> <a href='https://developer.mozilla.org/docs/Web/CSS'>CSS</a> property sets an element's inner and outer display types. The outer type sets an element's participation in <a href='https://developer.mozilla.org/docs/Web/CSS/CSS_Flow_Layout'>flow layout</a>; the inner type sets the layout of children."
```

Comment:
I struggled revising this one. I very much welcome suggestions here.

Reply:
Hmm, do you like:

The CSS display property sets whether an element is treated as a <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flow_Layout">block or inline element</a> and the layout used for its children, such as <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout">grid</a> or <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout">flex</a>.

Reply:
Unfortunately that is tight against the word limit so it's hard to make it better. But the idea being that display can work in one (or sometimes both) of two modes: to set the way this element participates in layout (block/inline) and to set the layout method for its children (e.g. flex/grid). Which your text has of course, but it seems helpful to make it concrete by mentioning things like block/inline and flex/grid rather than more abstract terms like "outer type" and "inner type".

As it happens I was looking at this today: https://www.smashingmagazine.com/2019/04/display-two-value/.

Reply:
Ah your text is so much better (and does pass our requirements). It must help to have an actual clue about CSS. 😆 

I moved the display type jargon to the following paragraph on the wiki page, since it's useful for reading the rest of the page.